### DISCUSSION TITLE
Pull Payments not settling or changing status

### DISCUSSION THREAD

**Discussion by bigsk1** [btcpayserver/btcpayserver#4494] [Technical Support] [answered]
Maybe I am doing something wrong or is a bug. Running BTCPay Server v1.7.3.0+09d5f5a08 - LunaNode docker deployment. 

Created a pull payment (edited)

Add btc on chain address and claimed, on backend Pull Payments / Payouts / Awaiting Payment / Action - Send selected payouts. I then send the transaction like normal. (edited screenshot)

The transaction is confirmed  (edited)

but the pull payment stays at "Awaiting Payment" in admin and on the front end still will show Awaiting Payment. 

I assume it should immediatly once funds are sent go to the "in progress" status and then once a certain number of confirmations are done go to "Completed" 

I have tried this a few times over the last few days and issue is same on all pull payments made.

**Comment by Kukks** [comment, marked as answer]
Did you by any chance use subtract form fee feature in the wallet send
feature? That would make the sent amount not equal to the payout and cause
the system to not match the payment.

**Comment by bigsk1** [comment]
Bingo! Just tried another test transaction and yep that was it, it went to progress right away. Thank you

**Reply by bigsk1** [reply]
One thing I should note I have tried another pull payment and when paying if I select subtract fee but also if I tick don't create utxo change the payment will be more than a $5 claim.
I made the mistake of sending $140 and it still doesn't go to processing and complete. So it appears if the subtract fee box is ticked at all even when paying a good amount over ($135) because of no utxo change it doesn't change status.
